_sass/minimal-mistakes/_footer.scss

```diff
@@ -9,7 +9,7 @@
   margin-right: 0;
   width: 100%;
   margin-top: 3em;
-  color: $muted-text-color;
+  color: #6c666d;
```

Comment:
@willingc this is great - given this is a variable that is used elsewhere likely and based on the text color, what do you think about modifying it [here](https://github.com/pyOpenSci/pyopensci.github.io/blob/main/_sass/minimal-mistakes/_variables.scss#L92)

and essentially changing the % text color to a bigger value?

$muted-text-color: mix(#fff, $text-color, 20%) !default;

you could locally see what the rgb value is or hex in the "computed" part of the web developer tools if you are using those. but maybe bump it up to 30 or 40% or whatever makes lighthouse / accessibility checks happy.

Reply:
Good idea!